schemas/v1/13_actor_events.go

```diff
@@ -0,0 +1,22 @@
+package v1
+
+func init() {
+	patches.Register(
+		13,
+		`
+	CREATE TABLE IF NOT EXISTS {{ .SchemaName | default "public"}}.actor_events (
+	    height 			bigint 	NOT NULL,
+	    state_root 		text 	NOT NULL,
+	    event_index 	bigint 	NOT NULL,
+	    message_cid 	text	NOT NULL,
+	    
+	    emitter 		text	NOT NULL,
+	    flags 			bytea	NOT NULL,
+	    key 			text 	NOT NULL,
+	    value			bytea	NOT NULL,
+	    
+		PRIMARY KEY ("height", "state_root", "event_index", "message_cid")
+	);
```

Comment:
@frrist is this tracking the new event schema which includes the codec? FIP-0049 was updated ~2 weeks ago to add a codec field (value 0x55, RAW) always.

Reply:
Will add the codec value shortly, missed that in my initial implementation. Thanks for the pointer!